src/sentry/middleware/__init__.py

```diff
@@ -23,7 +23,7 @@ def get_path(view_func: ViewFunc) -> str | None:
 
 
 def is_frontend_request(request: Request) -> bool:
-    return bool(getattr(request, "COOKIES", {}))
+    return bool(getattr(request, "COOKIES", {})) and getattr(request, "auth", None) is None
```

Comment:
put a docstring here to explain why this makes something a UI request, and how much this metric should be trusted